src/sql/workbench/parts/charts/browser/actions.ts

```diff
@@ -152,7 +152,11 @@ export class SaveImageAction extends Action {
 
 	public async run(context: IChartActionContext): Promise<boolean> {
 		if (context.insight instanceof Graph) {
-			const filePath = await this.fileDialogService.pickFileToSave({});
+			let fileFilters = new Array<FileFilter>();
+			let fileFilter: { extensions: string[]; name: string } = { extensions: ['png'], name: localize('resultsSerializer.saveAsFileExtensionPNGTitle', "PNG") };
```

Comment:
Could you just create the array with the item already in it?

Reply:
Sure